src/containers/Accounts/AMM/AMMAccounts/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { useLanguage } from '../../../shared/hooks'
+import '../../styles.scss'
+import formatBalance from '../../../../rippled/lib/txSummary/formatAmount'
+import {
+  getAccountTransactions,
+  getAMMInfo,
+} from '../../../../rippled/lib/rippled'
+import { Tabs } from '../../../shared/components/Tabs'
+import { analytics, ANALYTIC_TYPES, formatAsset } from '../../../shared/utils'
+import SocketContext from '../../../shared/SocketContext'
+import { ERROR_MESSAGES } from '../../Errors'
+import NoMatch from '../../../NoMatch'
+import {
+  AMMAccountHeader,
+  AmmDataType,
+} from './AMMAccountHeader/AMMAccountHeader'
+import { AccountTransactionTable } from '../../AccountTransactionTable'
+
+const getErrorMessage = (error: string) =>
+  ERROR_MESSAGES[error] || ERROR_MESSAGES.default
+
+function findAMMCreate(txs: [any]) {
+  const ammCreate = txs.filter((tx) => tx.tx.TransactionType === 'AMMCreate')
+
+  if (ammCreate.length < 1) throw new Error('Could not find AMM Create')
+
+  return ammCreate[0].tx
+}
+
+export const AMMAccounts = () => {
+  const { id: accountId, tab = 'transactions' } = useParams<{
+    id: string
+    tab: string
+  }>()
+  const { path = '/' } = useRouteMatch()
+  const { t } = useTranslation()
+  const mainPath = `${path.split('/:')[0]}/${accountId}`
+  const rippledSocket = useContext(SocketContext)
+  const [data, setData] = useState<AmmDataType>()
+  const [error, setError] = useState<any>()
+  const language = useLanguage()
+
+  useEffect(() => {
+    let asset1: { currency: string; issuer?: string }
+    let asset2: { currency: string; issuer?: string }
+    let ammData: any
+
+    /*
+    Get the first account transaction which in this case should be AMMCreate. From this we get
+    the two assets in the asset pool.
+    */
+    getAccountTransactions(rippledSocket, accountId, 1, undefined, true)
+      .then((tData) => {
+        const tx = findAMMCreate(tData.transactions)
+        asset1 = formatAsset(tx.Amount)
+        asset2 = formatAsset(tx.Amount2)
+
+        // if one of the assets is XRP, make sure it's the second one
+        if (asset1.currency === 'XRP') {
+          const temp = asset2
+          asset2 = asset1
+          asset1 = temp
+        }
+
+        return getAMMInfo(rippledSocket, asset1, asset2)
+      })
+
+      /*
+      Use the assets to get the AMM Info.
+      */
+      .then((ammDataWrapper) => {
+        ammData = ammDataWrapper.amm
+        const balance = formatBalance(ammData.Amount)
```

Comment:
The latest devnet update changed these properties.

Reply:
updated